test.py

```diff
@@ -416,9 +416,72 @@ class IntegerQueueArray(QueueArray[int]):
         self.assertRaises(Exception, test_qa.enqueue, 10)
         
     def test_queue_double_stack(self):
-        test_ds = StoreLineDoubleStack()
+        test_ds = StoreLineDoubleStack(size=1_000_000)
```

Comment:
I don't think the DoubleStack needs an initial size. I think only the array implementation requires an initial size